portfolio/src/main/webapp/index.html

```diff
@@ -2,56 +2,275 @@
+                    <p class="card-text" style="font-family:Monaco;font-size:22px">University of California San Diego (UCSD)</p>
+                    <p class="card-text" style="font-family:Monaco;font-size:22px">Major: Applied Mathematics</p>
+                    <p class="card-text" style="font-family:Monaco;font-size:22px">Double Major: Data Science</p>
+                    <p style="font-family:Monaco;font-size:20px">Contact Information at the end!</p>
+                </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="card mb-4" style="max-width: 1200px;">
+        <div class="row g-0">
+          <div class="col-md-8">
+            <div class="card-body">
+              <h5 style="font-family:Monaco;font-size: 30px;">CODING LANGUAGE:</h5>
+              <p></p>
+              <p style="font-family:Rockwell;">
+                <button type="button" class="btn btn-outline-secondary" style="font-size:25px">Python</button>
+                <button type="button" class="btn btn-outline-secondary" style="font-size:25px">Java</button>
+                <button type="button" class="btn btn-outline-secondary" style="font-size:25px">HTML</button>
+                <button type="button" class="btn btn-outline-secondary" style="font-size:25px">Javascript</button>
+                <button type="button" class="btn btn-outline-secondary" style="font-size:25px">C</button>
+              </p>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <img style="float:right;" src="./images/coding.jpeg" class="img-fluid rounded-start" alt="Coding">
+          </div>
+        </div>
+      </div>
+      <h3 style="font-family:Rockwell;">PROJECT</h3>
+      <h5 style="font-family:Rockwell;">Click each bottum to see more details of each project</h5>
+        <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#Modal1" style="font-family:Comic Sans MS;">
+            Course Project (Java)
+        </button>
+        <div class="modal fade" id="Modal1" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+            <div class="modal-dialog modal-dialog-centered">
+                <div class="modal-content">
+                    <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel" style="font-family:Rockwell;">Life of Cell Project</h5>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                    </div>
+                    <div class="modal-body" style="font-family:'Times New Roman', Times, serif;">
+                        Create Cell classes and a PetriDish class that 
+                        represents a 2D board of different types of cells to move and play the game
+                    </div>
+                    <div class="modal-body; center">
+                        To be display soon...
```

Comment:
Maybe remove this and add a todo comment instead. This lets you leave a reminder for yourself without showing the user that there is missing content.


```html
<!-- Add project display information -->
```